pkgs/servers/sql/postgresql/ext/timescaledb_toolkit.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildPgxExtension
+, postgresql
+, stdenv
+, nixosTests
+}:
+
+buildPgxExtension rec {
+  inherit postgresql;
+
+  pname = "timescaledb_toolkit";
+  version = "1.13.1";
+
+  src = fetchFromGitHub {
+    owner = "timescale";
+    repo = "timescaledb-toolkit";
+    rev = version;
+    sha256 = "sha256-sds2q7wS9uxd89n6CPLnn+Eiw3Y0soqRCXnbaVytho0=";
+  };
+
+  cargoSha256 = "sha256-YyG/eMumHWu5KR3Tqe2VIwLRUvvKPOc6An4yR46W0f4=";
+  buildAndTestSubdir = "extension";
+
+  passthru.tests = {
+    timescaledb_toolkit = nixosTests.timescaledb;
+  };
+
+  # tests take really long
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Provide additional tools to ease all things analytic when using TimescaleDB";
+    homepage = "https://github.com/timescale/timescaledb-toolkit";
+    maintainers = with maintainers; [ typetetris ];
+    platforms = postgresql.meta.platforms;
+    license = licenses.asl20;
+
+    # as it needs to be used with timescaledb, simply use the condition from there
+    broken = versionOlder postgresql.version "12";
```

Comment:
If that is essentially copied, would it make sense to use timescaledb.meta.broken?

Reply:
`timescaledb` and `timescaledb-toolkit` have different version schemes. So it might happen, the latest version of `timescaledb` does support a new postgresql version, but the latest version of `timescaldb-toolkit` doesn't. And the other way round. But yes, it will most likely be short periods of time.

Reply:
I would need to change (in `pkgs/servers/sql/postgresql/packages.nix`)
`timescaledb_toolkit = super.callPackage ./ext/timescaledb_toolkit.nix { };`
to
`timescaledb_toolkit = self.callPackage ./ext/timescaledb_toolkit.nix { };`
to get access to `timescaledb` and it would be the first extension to do so.

So I would like to avoid that. A tiny bit of duplication is better than an additional
dependency in my mind.